dali/python/backend_impl.cc

```diff
@@ -864,6 +864,15 @@ py::dict MetaToDict(const ReaderMeta &meta) {
   return d;
 }
 
+template <typename Backend>
+void FeedPipeline(Pipeline *p, const string &name, py::list list, cudaStream_t stream) {
+  vector<Tensor<Backend>> tensors(list.size());
+  for (size_t i = 0; i < list.size(); ++i) {
+    tensors[i] = std::move(list[i].cast<Tensor<Backend>&>());
+  }
+  p->SetExternalInput(name, tensors, stream);
```

Comment:
Since you're creating this vector by hand, you can create a TensorVector as well and avoid extra conversion step.